projects/index.md

```diff
@@ -9,59 +9,62 @@ In general, projects should have:
 
-Since most of the [hubs we run](https://pilot-hubs.2i2c.org/en/latest/reference/hubs.html) are
-managed out of the [2i2c-org/pilot-hubs](https://github.com/2i2c-org/pilot-hubs) repository,
-deliverables are tracked in that repository as well.
+:::{admonition} Tracking deliverables
+- [the high level goals and strategy are described here](https://pilot.2i2c.org/en/latest/about/strategy.html)
+- [the roadmap for the service is described here](https://pilot.2i2c.org/en/latest/about/roadmap.html)
+:::
 
 (projects:jmte-pangeo)=
 ## Pangeo Hub Infrastructure development
 
 The Pangeo project has considerable needs in both operating and developing hub infrastructure for their community.
 We are collaborating with them to provide both daily operations of their hubs, as well as help them in evolvin and improving their infrastructure to better-suit their scientific mission.
 
-**Tracking deliverables**
-
-This project is just beginning, and currently does not have a location to track its deliverables.
-However, you can find discussion of general Pangeo activity in [the Pangeo repository](https://github.com/pangeo-data/pangeo).
-
-## Jupyter Meets the Earth
+:::{admonition} Tracking deliverables
 
-Jupyter Meets the Earth is an NSF EarthCube project to develop Jupyter infrastructure in service of the earth sciences.
+**Note that this initiative is WIP and may change in scope and focus**
 
-**Tracking deliverables**
-
-The [`pangeo-data/jupyter-earth` repository](https://github.com/pangeo-data/jupyter-earth) contains deliverables for this project. Look for [the {guilabel}`🏷 JupyterHub` label](https://github.com/pangeo-data/jupyter-earth/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22%3Alabel%3A+JupyterHub%22).
+We are tracking deliverables for this project [in this GitHub project board](https://github.com/orgs/2i2c-org/projects/16).
+:::
 
 ## Jupyter Book development
 
 The Executable Books Project is an international collaboration to build open source tools that facilitate publishing computational narratives using the Jupyter ecosystem.
 Its activities are split across a variety of repositories in the [`executablebooks/` organization](https://github.com/executablebooks).
 
-**Tracking deliverables**
+:::{admonition} Tracking deliverables
 
 - The [`meta/` repository](https://github.com/executablebooks/meta) contains high-level conversation and strategy in the project. Most project planning and execution is carried out in the issues in `executablebooks/` repositories.
 - The [`jupyter-book` repository](https://github.com/executablebooks) contains many issues that are the best place to get started finding things of interest to work on.
 - The [`executablebooks` feature voting board](https://executablebooks.org/en/latest/feature-vote.html) contains a table of issues users have "voted" for. It is a good guide for prioritizing work.
+
+:::
+
+```{toctree}
+managed-hubs/index
+```
```

Comment:
About this toctree, should not be at the end of the "Managed Hub Service launch"?
Looking at it now, it appears under the JupyterBook stuff and seems disconnected.

Reply:
good point @damianavila - updated the location